WORKSPACE

```diff
@@ -33,6 +33,18 @@ bazel_skylib_workspace()
 load("@bazel_skylib//lib:versions.bzl", "versions")
 versions.check(minimum_bazel_version = "6.0.0")
 
+# RapidJSON
+# Must be defined earlier than tensorflow_serving because TFS is using older rapidjson
+# Version must match openvino.genai -> jinja2cpp -> rapidjson
+# git/Jinja2Cpp/thirdparty/internal_deps.cmake
+http_archive(
+    name = "com_github_tencent_rapidjson",
+    url = "https://github.com/Tencent/rapidjson/archive/973dc9c06dcd3d035ebd039cfb9ea457721ec213.tar.gz",
```

Comment:
Which version is it? Maybe we should add to comment

Reply:
sha before .tar.gz is sha of the commit.